### ISSUE TITLE
WIP: Github Vulnerability scan import serialization issue

### ISSUE THREAD

**Issue by hex0d** [DefectDojo/django-DefectDojo#2984]
**Bug description**
https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/tools/github_vulnerability/parser.py#L75U
Using Github Vulnerability Scan while the setting deduplicate finding is turned on get an error 500 and the docker-compose logs present the error "cannot resialize '_io.BufferedRandom'" and as pointed by @valentijnscholten "the filereader is attached to the finding which celery is trying to serialize."

**Steps to reproduce**
1. Go to Your Dojo Configuration -> System Settings
2. Turn On Deduplicate findings and Click Submit on the bottom of the screen
3. Post to "/api/v2/import-scan/" the data in sample scan file (in this document):

4. You Should get a 500 and a message on docker-compose logs "cannot resialize '_io.BufferedRandom"
5. Turn off Deduplication
6. Repeat step 3, and it should work fine

**Expected behavior**
Get a HTTP 500 and a message on docker-compose logs "cannot resialize '_io.BufferedRandom".

**Deployment method**
Docker

**Environment information**
 - Operating System:  Ubuntu 18.04
 - DefectDojo Commit Message: Latest

**Sample scan files** (optional)

```
{
    "data": {
      "search": {
        "repositoryCount": 13,
        "pageInfo": {
          "endCursor": "xxxx",
          "hasNextPage": true
        },
        "nodes": [
          {
            "name": "xxxx",
            "vulnerabilityAlerts": {
              "pageInfo": {
                "endCursor": "yyyyyyy",
                "hasNextPage": true
              },
              "nodes": [
                {
                  "dismissedAt": "2019-10-22T10:41:21Z",
                  "id": "xxxxxxxxx",
                  "securityVulnerability": {
                    "package": {
                      "name": "xxxxxxxx"
                    },
                    "advisory": {
                      "description": "XXXXXXXXXXXX",
                      "summary": "XXXXXXXXXXXXXXX",
                      "references": [
                        {
                          "url": "CXCXCXCXCXXCCXXCCXXCCXXCCXXCCX"
                        },
                        {
                          "url": "VCCVVCXVCXVXZCVZVZCXVXZVCXZVZXVCXZVXCC"
                        }
                      ],
                      "identifiers": [
                        {
                          "value": "DSFAFSDFASFDSAFASD"
                        },
                        {
                          "value": "ADFSFDSAFASD"
                        }
                      ]
                    },
                    "severity": "MODERATE"
                  }
                },
                {
                  "dismissedAt": "2020-04-01T19:34:55Z",
                  "id": "AFDSFSDAFSDASFDAFSDASFD=",
                  "securityVulnerability": {
                    "package": {
                      "name": "ASFDSAFD"
                    },
                    "advisory": {
                      "description": "ASFDSVCZXCASDFVZXC",
                      "summary": "AMSVNASCMASNCADNNJSADC",
                      "references": [
                        {
                          "url": "CDAJNDSKJAKJNCDASJCANSK"
                        },
                        {
                          "url": "DCAKLSCDKNLDKLDASNLKCADSNCLADKSNDS"
                        },
                        {
                          "url": "DCAKNLSNDKCKLDASNKLDASNCKLNCA"
                        }
                      ],
                      "identifiers": [
                        {
                          "value": "DACOIJASOIDACKPOCP"
                        },
                        {
                          "value": "CDAOPKACKADCIJCAIJOCDIO"
                        }
                      ]
                    },
                    "severity": "MODERATE"
                  }
                },
                {
                  "dismissedAt": "2020-04-01T19:32:54Z",
                  "id": "DASFSDAFASDFADFSASDFAFSDDFAS=",
                  "securityVulnerability": {
                    "package": {
                      "name": "ADFS"
                    },
                    "advisory": {
                      "description": "FADSFSDDSADAGSGDSFAFDAS",
                      "summary": "OSDPSAOKFIOASDJOFIASDPKFOASDNJFAOSKPFDKNJASPF",
                      "references": [
                        {
                          "url": "AKDFMCKSNKASDNJKCDIAJSONJKCAJSOPDCS"
                        },
                        {
                          "url": "SDACNJIAONCAISCNDSICAON"
                        }
                      ],
                      "identifiers": [
                        {
                          "value": "DSACNKAJODANDSACKJOC"
                        },
                        {
                          "value": "DASDCDAADLADSKMNMDKL"
                        }
                      ]
                    },
                    "severity": "MODERATE"
                  }
                },
                {
                  "dismissedAt": "2020-04-01T19:32:54Z",
                  "id": "DASFMMFKLNKDSAKFSDLANJKKFDSNJSAKDFNJKDFS=",
                  "securityVulnerability": {
                    "package": {
                      "name": "FDNSALSKDFLANKKSDFJLNKDFSK"
                    },
                    "advisory": {
                      "description": "ASDBHJSANKJHSKADNSABDJNKSAKJDAe",
                      "summary": "SDKPKÁSMNMKSDANJDOPASJOKNDOSAJ",
                      "references": [
                        {
                          "url": "MSADKDKSPAIJONIDAPSJIOSPDAOKDSPAKDPOSKPDOSAOKSAD"
                        },
                        {
                          "url": "SADMKASJDONIAPSJODNIPAJSIONDPKOASD"
                        },
                        {
                          "url": "SAPOSDOKJSKADOPDSAJOKOSDAPJSAPJSDAJPSODAPODSJ"
                        }
                      ],
                      "identifiers": [
                        {
                          "value": "ASDLKSDANNKJPDKNALJOPDSJIOAN"
                        },
                        {
                          "value": "ASDKNLJASJDPKNKJDOSIA0SAKMSDAP"
                        }
                      ]
                    },
                    "severity": "MODERATE"
                  }
                }
              ]
            }
          },
          {
            "name": "ASDKNJSADJNSADPJODSNOJIASON",
            "vulnerabilityAlerts": {
              "pageInfo": {
                "endCursor": null,
                "hasNextPage": false
              },
              "nodes": []
            }
          },
          {
            "name": "SMADPKDSOKPODKSAPSDKONOSAJD",
            "vulnerabilityAlerts": {
              "pageInfo": {
                "endCursor": null,
                "hasNextPage": false
              },
              "nodes": []
            }
          },
          {
            "name": "SADMPMKDNSOPONDSAOPKONSDAPKS",
            "vulnerabilityAlerts": {
              "pageInfo": {
                "endCursor": null,
                "hasNextPage": false
              },
              "nodes": []
            }
          }
        ]
      }
    }
  }
```

**Comment by damiencarol** [comment]
@valentijnscholten will take a llok to see if it's till there.

**Comment by damiencarol** [comment]
fixed in `dev` and confirmed it works on `1.14.1`
